magento/2/latest-version

```diff
@@ -1 +1 @@
-2.4.6-p4
\ No newline at end of file
+2.4.6-p4
```

Comment:
Should this change be here? Pretty sure we patched this to p5 earlier this week

Reply:
is that a pending PR? master has 2.4.6-p4 in this file https://github.com/nexcess/app-sources/blob/master/magento/2/latest-version

Reply:
Wait, now I'm not sure - is this repo just irrelevant for Magento?